docker/azure-services/local-webapp-env-template

```diff
@@ -14,3 +14,4 @@ export MAX_GEOSPATIAL_LAND_BOUNDARY_UPLOAD_MB="50"
 # Change to http://azure_services:7071/api if the application to register web application is running in a container.
 export SIGNALR_URL="http://azure_services:8082/api"
 export MAX_METRIC_UPLOAD_MB="50"
+export GOOGLE_TAGMANAGER_ID="GTM-PL85H6J"
```

Comment:
For local development purposes app should accept a blank tagmanager ID and skip its loading, so as not to send data to GTM